test/smoke/steps/graphql_requests.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2020 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package steps
```

Comment:
I didn't want to mix it with the file declaring available steps, to keep that file easily readable and dedicated just to one purpose (declare and implement Cucumber steps). Though if you think otherwise I can move the request.

Reply:
should we then put that into the framework instead ? I mean the whole ExecuteGraphQLRequest and then analyze the result here.
or should we put query string and response struct together in the same file ?

This is not a huge thing for now. 
Could be refactored later if it aims to grow bigger.

btw the way, your query is asking for only `processName` but in the response, you are expecting more (`id` and `processId`). Is that on purpose ?

Reply:
> should we then put that into the framework instead ?

The reason I put it here is that this implementation is specifically related to Process Instances. From my point of view the framework should be kept generic to process any GraphQL request, not bound to specific implementation.
I hope I am not overengineering it :) 

> or should we put query string and response struct together in the same file ?

Good point, will move them to be together.

> btw the way, your query is asking for only processName but in the response, you are expecting more (id and processId). Is that on purpose ?

It is leftover from the implementation, when I consider what to return. Will remove the unused parameters.

Reply:
> The reason I put it here is that this implementation is specifically related to Process Instances. From my point of view the framework should be kept generic to process any GraphQL request, not bound to specific implementation.

Seems a good point :)

Let's keep them together in the separate file then.